### PR TITLE
Clarify the supported Terraform version for prep_for_subm

### PR DESCRIPTION
**Pull request by nyechiel**
Signed-off-by: nyechiel <nyechiel@redhat.com>

### CODE REVIEW

File: src/content/quickstart/openshift/_index.md [hunk 1]
```diff
@@ -26,6 +26,8 @@ please refer to the official [OpenShift documentation](https://docs.openshift.co
 
 {{% notice info %}}
 Please note that  `oc`, `aws-cli`, `terraform`, and `wget` need to be installed before the `prep_for_subm.sh` script can be run.
+Also note that the script is known to be working with [Terraform](https://releases.hashicorp.com/terraform/) version 0.12.
```
Comment (sridhargaddam) [inline src/content/quickstart/openshift/_index.md:29]: Looks good. 
Just a thought.
Shall we say that the script uses Terraform internally and the max supported version is 0.12.12

Comment (nyechiel) [reply src/content/quickstart/openshift/_index.md:29]: @sridhargaddam it says that the "maximum compatible version is 0.12.12" - isn't it clear enough?

Comment (sridhargaddam) [reply src/content/quickstart/openshift/_index.md:29]: It is :) 
Just wanted to highlight that the script internally uses Terraform.

**Review by sridhargaddam — APPROVED** [review]

**Comment by dfarrell07** [comment]
Seems like https://github.com/submariner-io/submariner/pull/949 suggests a different supported version? @manosnoam

File: src/content/quickstart/openshift/_index.md (hunk 1 above)
Comment (sridhargaddam) [reply src/content/quickstart/openshift/_index.md:29]: Looks like @manosnoam tested with 0.12.29 and it worked. 
https://github.com/submariner-io/submariner/pull/949

**Comment by nyechiel** [comment]
> Seems like [submariner-io/submariner#949](https://github.com/submariner-io/submariner/pull/949) suggests a different supported version? @manosnoam

We have seen some inconsistent behaviour with different versions of Terraform, so the idea is to limit the versions /cc @mangelajo

**Review by mangelajo — APPROVED** [review]

File: src/content/quickstart/openshift/_index.md (hunk 1 above)
Comment (manosnoam) [reply src/content/quickstart/openshift/_index.md:29]: Right, on Terraform v0.12.29 it works good, I've verified again.

**Pull request merged**